source/mixins/versionable.js

```diff
@@ -0,0 +1,19 @@
+Space.messaging.Versionable = {
+
+  schemaVersion: 1,
+
+  onConstruction(data) {
+    if (_.isObject(data)) this._migrateToLatestVersion(data);
```

Comment:
@rhyslbw do you mean that there should only be a single transform method? or `transformFromVersionX` for each version like now?